glib/src/thread_guard.rs

```diff
@@ -111,4 +112,40 @@ impl<T> Drop for ThreadGuard<T> {
     }
 }
 
+impl<T: Default> Default for ThreadGuard<T> {
+    fn default() -> Self {
+        Self::new(T::default())
+    }
+}
+
 unsafe impl<T> Send for ThreadGuard<T> {}
+
+impl<T: crate::ToValue + crate::StaticType> crate::ToValue for ThreadGuard<T> {
+    fn to_value(&self) -> crate::Value {
+        T::to_value(self.get_ref())
+    }
+
+    fn value_type(&self) -> crate::Type {
+        T::static_type()
+    }
+}
+
+unsafe impl<'a, T, C, E> crate::value::FromValue<'a> for ThreadGuard<T>
```

Comment:
should probably import all the `crate::value::` types

Reply:
:shrug: